src/Pact/Repl/Lib.hs

```diff
@@ -460,16 +460,24 @@ verify i as = case as of
     md <- getModule i (ModuleName modName Nothing)
     -- reading all modules from db here, but should be fine in repl
     modules <- getAllModules i
+    let failureMessage = tStr $ "Verification of " <> modName <> " failed"
 #if defined(ghcjs_HOST_OS)
     uri <- fromMaybe "localhost" <$> viewLibState (view rlsVerifyUri)
     renderedLines <- liftIO $
                      RemoteClient.verifyModule modules md uri
+    setop $ TcErrors $ Text.unpack <$> renderedLines
+    return failureMessage
```

Comment:
Also upon merging master into this branch, I assumed that per the motivation in b6c5d4ae we would want to display "Verification of [module] failed" here too.

Reply:
I'm confused about this point since we return `failureMessage`

Reply:
There was a merge conflict, and I changed the code so that we now return "Verification failed" in both cases -- not just the single case in which we were doing this before the merge.

Reply:
This is perfect -- thanks!